From 0ec5c81db656d5e161590cd884de17cd794b253c Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sun, 30 Jun 2024 17:25:00 +0800
Subject: [PATCH 25/30] Update

---
 libbb/appletlib.c | 3 ---
 libbb/messages.c  | 2 +-
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/libbb/appletlib.c b/libbb/appletlib.c
index 4e24ae4..7cd9984 100644
--- a/libbb/appletlib.c
+++ b/libbb/appletlib.c
@@ -780,9 +780,6 @@ int busybox_main(int argc UNUSED_PARAM, char **argv)
 		full_write2_str(bb_banner); /* reuse const string */
 		full_write2_str(" multi-call binary.\n"); /* reuse */
 		full_write2_str(
-			"BusyBox is copyrighted by many authors between 1998-2015.\n"
-			"Licensed under GPLv2. See source distribution for detailed\n"
-			"copyright notices.\n"
 			"\n"
 			"Usage: busybox [function [arguments]...]\n"
 			"   or: busybox --list"IF_FEATURE_INSTALLER("[-full]")"\n"
diff --git a/libbb/messages.c b/libbb/messages.c
index 6914d57..4fd81c9 100644
--- a/libbb/messages.c
+++ b/libbb/messages.c
@@ -11,7 +11,7 @@
 #define BB_EXTRA_VERSION " ("AUTOCONF_TIMESTAMP")"
 #endif
 
-const char bb_banner[] ALIGN1 = "BusyBox v" BB_VER BB_EXTRA_VERSION;
+const char bb_banner[] ALIGN1 = "Enhanced BusyBox v" BB_VER BB_EXTRA_VERSION;
 
 
 const char bb_msg_memory_exhausted[] ALIGN1 = "out of memory";
-- 
2.45.2

