From 246a0439ddd6f7f339ee4e4fb2ba68c5584ee4b4 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sun, 30 Jun 2024 21:34:40 +0800
Subject: [PATCH 27/31] test

---
 libbb/messages.c | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/libbb/messages.c b/libbb/messages.c
index 4fd81c9..d1039af 100644
--- a/libbb/messages.c
+++ b/libbb/messages.c
@@ -6,12 +6,15 @@
  */
 #include "libbb.h"
 
-/* allow version to be extended, via CFLAGS */
-#ifndef BB_EXTRA_VERSION
-#define BB_EXTRA_VERSION " ("AUTOCONF_TIMESTAMP")"
+/* timestamp */
+#define BB_EXTRA_VERSION AUTOCONF_TIMESTAMP
+
+/* define builder, if not defined */
+#ifndef BB_BUILDER
+#define BB_BUILDER "nobody@ydobon"
 #endif
 
-const char bb_banner[] ALIGN1 = "Enhanced BusyBox v" BB_VER BB_EXTRA_VERSION;
+const char bb_banner[] ALIGN1 = "Enhanced BusyBox v" BB_VER " ("BB_BUILDER")" " ("BB_EXTRA_VERSION")" "\n";
 
 
 const char bb_msg_memory_exhausted[] ALIGN1 = "out of memory";
-- 
2.45.2

