From fbe5b074b469673baf8d8886cc1271d85918a1f4 Mon Sep 17 00:00:00 2001
From: eraselk <eraselk@proton.me>
Date: Fri, 26 Jul 2024 17:19:44 +0800
Subject: [PATCH 31/31] Fix some error

---
 libbb/hash_md5_sha.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libbb/hash_md5_sha.c b/libbb/hash_md5_sha.c
index 57a8014..c570694 100644
--- a/libbb/hash_md5_sha.c
+++ b/libbb/hash_md5_sha.c
@@ -1197,7 +1197,7 @@ void FAST_FUNC sha1_begin(sha1_ctx_t *ctx)
 		if (!ni)
 			ni = get_shaNI();
 		if (ni > 0)
-			ctx->process_block = sha1_process_block64_shaNI;
+			ctx->process_block = sha1_process_block64;
 	}
 # endif
 #endif
@@ -1313,7 +1313,7 @@ unsigned FAST_FUNC sha1_end(sha1_ctx_t *ctx, void *resbuf)
 	hash_size = 8;
 	if (ctx->process_block == sha1_process_block64
 #if ENABLE_SHA1_HWACCEL
-	 || ctx->process_block == sha1_process_block64_shaNI
+	 || ctx->process_block == sha1_process_block64
 #endif
 	) {
 		hash_size = 5;
-- 
2.45.2

